### ISSUE TITLE
Compute nodes inherit EPEL from controller but not its key.

### ISSUE THREAD

**Issue by rispoli** [clustervision/trinity#612]
```
[edin - root@controller(150373) ~]# ssh node002
[root@node002 ~]# yum install htop
Loaded plugins: fastestmirror
dell-system-update_dependent                             | 2.3 kB     00:00     
dell-system-update_independent                           | 2.3 kB     00:00     
epel/x86_64/metalink                                     |  26 kB     00:00     
epel                                                     | 4.3 kB     00:00     
xCAT-centos7-path0                                       | 3.6 kB     00:00     
xcat-otherpkgs0                                          | 2.9 kB     00:00     
(1/2): epel/x86_64/updateinfo                              | 473 kB   00:04     
(2/2): epel/x86_64/primary_db                              | 3.8 MB   00:35     
Loading mirror speeds from cached hostfile
 * epel: mirror.de.leaseweb.net
Resolving Dependencies
--> Running transaction check
---> Package htop.x86_64 0:1.0.3-3.el7 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package         Arch              Version                Repository       Size
================================================================================
Installing:
 htop            x86_64            1.0.3-3.el7            epel             87 k

Transaction Summary
================================================================================
Install  1 Package

Total download size: 87 k
Installed size: 181 k
Is this ok [y/d/N]: y
Downloading packages:
warning: /var/cache/yum/x86_64/7/epel/packages/htop-1.0.3-3.el7.x86_64.rpm: Header V3 RSA/SHA256 Signature, key ID 352c64e5: NOKEY
Public key for htop-1.0.3-3.el7.x86_64.rpm is not installed
htop-1.0.3-3.el7.x86_64.rpm                                |  87 kB   00:05     
Retrieving key from file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7


GPG key retrieval failed: [Errno 14] curl#37 - "Couldn't open file /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7"
```

Even trying to re-install the repo doesn't fix the problem:
```
[root@node002 ~]# yum install epel-release
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * epel: mirror.de.leaseweb.net
Resolving Dependencies
--> Running transaction check
---> Package epel-release.noarch 0:7-5 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package                 Arch              Version        Repository       Size
================================================================================
Installing:
 epel-release            noarch            7-5            epel             14 k

Transaction Summary
================================================================================
Install  1 Package

Total download size: 14 k
Installed size: 24 k
Is this ok [y/d/N]: y
Downloading packages:
warning: /var/cache/yum/x86_64/7/epel/packages/epel-release-7-5.noarch.rpm: Header V3 RSA/SHA256 Signature, key ID 352c64e5: NOKEY
Public key for epel-release-7-5.noarch.rpm is not installed
epel-release-7-5.noarch.rpm                                |  14 kB   00:05     
Retrieving key from file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7


GPG key retrieval failed: [Errno 14] curl#37 - "Couldn't open file /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7"
```

I ended up copying the key into the rootimg and re-packing it but it's not a good solution.
What would be a long term fix?

**Comment by rispoli** [comment]
$ cat cv_install_epel_on_compute 
wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
rpm -ivh epel-release-latest-7.noarch.rpm
rm -f epel-release-latest-7.noarch.rpm

**Comment by Marc69** [comment]
Resolved in * r8 3bb0b2d (tmaster)
